### DISCUSSION TITLE
does chrome fingerprinting works now?

### DISCUSSION THREAD

**Discussion by tserebour** [daijro/camoufox#112] [Fingerprinting]
just want to know if Chrome fingerprinting works now or still needs some work

**Comment by daijro** [comment]
Hello,

Unfortunately Chrome fingerprints won't be supported any time soon. 

Most anti-bots check if your JS engine is Chrome (v8) or Firefox (SpiderMonkey). This is impossible to spoof. There would need to be a separate Chrome custom build for Chrome anti-fingerprinting.
